clang/test/SemaSYCL/num_simd_work_items_device.cpp

```diff
@@ -42,66 +42,88 @@ struct FuncObj {
 };
 
 #ifdef TRIGGER_ERROR
-// If the declaration has an [[intel::reqd_work_group_size]] or
-// [[cl::reqd_work_group_size]] attribute, tests that check if
-// the work group size attribute argument (the first argument)
-// can be evenly divided by the num_simd_work_items attribute.
+// If the declaration has a [[intel::reqd_work_group_size]]
+// [[cl::reqd_work_group_size]] attribute, tests that check
+// if the work group size attribute argument (the last argument)
+// can be evenly divided by the [[intel::num_simd_work_items()]] attribute.
 struct TRIFuncObjBad1 {
   [[intel::num_simd_work_items(3)]]        // expected-error{{'num_simd_work_items' attribute must evenly divide the work-group size for the 'reqd_work_group_size' attribute}}
-  [[intel::reqd_work_group_size(5, 3, 3)]] // expected-note{{conflicting attribute is here}}
+  [[intel::reqd_work_group_size(3, 6, 5)]] // expected-note{{conflicting attribute is here}}
   void
   operator()() const {}
 };
 
 struct TRIFuncObjBad2 {
-  [[intel::reqd_work_group_size(5, 3, 3)]] // expected-note{{conflicting attribute is here}}
+  [[intel::reqd_work_group_size(3, 6, 5)]] // expected-note{{conflicting attribute is here}}
   [[intel::num_simd_work_items(3)]]        // expected-error{{'num_simd_work_items' attribute must evenly divide the work-group size for the 'reqd_work_group_size' attribute}}
   void
   operator()() const {}
 };
 
 struct TRIFuncObjBad3 {
-  [[intel::num_simd_work_items(3)]]     // expected-error{{'num_simd_work_items' attribute must evenly divide the work-group size for the 'reqd_work_group_size' attribute}}
-  [[cl::reqd_work_group_size(5, 3, 3)]] // expected-note{{conflicting attribute is here}}
+  [[intel::num_simd_work_items(3)]]  // expected-error{{'num_simd_work_items' attribute must evenly divide the work-group size for the 'reqd_work_group_size' attribute}}
+  [[intel::reqd_work_group_size(3)]] //expected-note{{conflicting attribute is here}}
```

Comment:
Can you add a FIXME comment here about the error being wrong -- this should be accepted once we fix the issue with the three different forms of the attribute (instead of assuming default values).

Reply:
Done